google/resource_container_cluster_test.go

```diff
@@ -12,6 +12,80 @@ import (
 	"github.com/hashicorp/terraform/terraform"
 )
 
+// TestContainerClusterNodeVersionCustomizeDiffFuncNilReturn is doing unit test for cases where
+// the return have to be "nil".
+func TestContainerClusterNodeVersionCustomizeDiffFuncNilReturn(t *testing.T) {
```

Comment:
nit: You can collapse both tables into one, and use a boolean in the test case to disambiguate whether you expect an error or not.